src/material/input/input.md

```diff
@@ -50,7 +50,7 @@ element. In some cases that `<mat-form-field>` may use the placeholder as the la
 The `<mat-form-field>` allows you to
 [associate error messages](https://material.angular.io/components/form-field/overview#error-messages)
 with your `matNativeControl`. By default, these error messages are shown when the control is invalid and
-either the user has interacted with (touched) the element or the parent form has been submitted. If
```

Comment:
I think the "either" here makes sense, but we should rephrase the sentence so it's easier to understand. Basically the logic is `control.invalid && (control.touched || parentForm.submitted)`.